src/draw/DefaultDrawer.js

```diff
@@ -166,7 +183,7 @@ class DefaultDrawer {
   }
 
   /**
-   * Unselect current selected element and select element by its id.
+   * Unselects current selected element and select element by its id.
```

Comment:
`and select` -> `and selects`

Reply:
Done

Reply:
Not done correctly

Reply:
Done correctly

Reply:
Indeed BUT you forgot to put back your first correction ! ;)